package/src/components/StockWarning/v1/StockWarning.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { applyTheme } from "../../../utils";
+
+const Span = styled.span`
+  display: block;
+  color: ${applyTheme("color_error")};
```

Comment:
Can you instead add `rui_stockWarningColor` to the default theme and use that? It's very possible people will want to change it to a color other than the error color.